docs/sections/user_guide/cli/tools/config.rst

```diff
@@ -175,6 +176,8 @@ In ``uw`` terminology, to realize a configuration file is to transform it from i
          Path to output file (defaults to stdout)
      --output-format {ini,nml,sh,yaml}
          Output format
+     --output-block KEY[:KEY:...]
```

Comment:
I wonder if `.` might be a better separator for this use case, since it would match Jinja2's syntax, e.g. `{{ a.b.c }}`. I think colons make sense as a path separator since they're used in the shell `PATH` environment variable. This was mentioned in the AC for [UW-559](https://jira.epic.oarcloud.noaa.gov/browse/UW-559), which is a similar use case, so maybe the use of colons here is an intentional change.

Reply:
Wow! We literally just discussed that one and I missed the dot-separation. I like it better and will update.